spring-boot-modules/spring-boot-1-actuator/.gitignore

```diff
@@ -0,0 +1,23 @@
+target/
```

Comment:
The root directory's .gitignore file should take care of this, so we can delete this file.